tilelang/__init__.py

```
@@ -142,7 +142,7 @@ def _load_tile_lang_lib():
     if env.SKIP_LOADING_TILELANG_SO == "0":
         _LIB, _LIB_PATH = _load_tile_lang_lib()
 
-    from .jit import jit, lazy_jit, JITKernel, compile, par_compile  # noqa: F401
+    from .jit import jit, JITKernel, compile, par_compile  # noqa: F401
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)